oms-kemp-applicationdelivery-solution/azuredeploy.json

```diff
@@ -719,8 +717,8 @@
         {
             "type": "Microsoft.OperationsManagement/solutions",
             "name": "[variables('omsSolutions').customSolution.solutionName]",
-            "apiVersion": "[variables('omsLogAnalyticsAPIVersion')]",
-            "location": "[variables('location')]",
+            "apiVersion": "2015-11-01-preview",
+            "location": "[parameters('omsLogAnalyticsRegion')]",
             "dependsOn": [
                 "[resourceId('Microsoft.OperationalInsights/workspaces/', parameters('omsLogAnalyticsWorkspaceName'))]",
                 "[concat('Microsoft.OperationalInsights/workspaces/', parameters('omsLogAnalyticsWorkspaceName'), '/views/', variables('omsSolutions').customSolution.viewName)]",
```

Comment:
is there a particular reason why you are mixing resourceId and concat in the dependsOn section? Recommend to stick to one approach :-) for the view, the resourceId will be [resourceId('Microsoft.OperationalInsights/workspaces/views/', parameters('omLogAnalyticsWorkspaceName'), variables('omsSolutions').customSolution.viewName))]